lib/Gitlab/Api/Projects.php

```diff
@@ -496,11 +496,20 @@ public function removeLabel($project_id, $name)
 
     /**
      * @param int $project_id
+     * @param array $params (
+     *
+     *     @var string $namespace      The ID or path of the namespace that the project will be forked to
+     * )
      * @return mixed
      */
-    public function fork($project_id)
+    public function fork($project_id, array $params = array())
     {
-        return $this->post('projects/'.$this->encodePath($project_id).'/fork');
+        $resolver = $this->createOptionsResolver();
```

Comment:
You can use `$this->createOptionsResolver()` here as this methods returns an `OptionsResolver` preconfigured for pagination. Here a `new OptionsResolver()` will do the job.

Reply:
It looks like you missed this one ;)